bot/handlers/bot_message_handler.py

```diff
@@ -31,13 +34,24 @@ def register(self, dp: Dispatcher) -> None:
         dp.message.register(self.handle, Command(commands=self.get_commands()))
 
     async def handle(self, message: Message) -> None:
+
         await self._log_user_activity(message.from_user.id, message.text)
+
+        if (
+            not await DatabaseManager.is_admin_or_moderator(message.from_user.id)
+            and await DatabaseManager.is_command_limited(message.from_user.id, settings.MESSAGE_LIMIT, settings.LIMIT_DURATION)
+        ):
+            await message.answer(get_limit_exceeded_message())
+            return
```

Comment:
kurde no nie podoba mi sie to miejsce, to jest doslownie cos co powinno middleware zalatwiac